sonar-project.properties

```diff
@@ -1 +1 @@
-sonar.exclusions = infraohjelmointi_api/**/tests/**/*, infraohjelmointi_api/**/migrations/**/*
\ No newline at end of file
+sonar.exclusions = infraohjelmointi_api/**/tests/**/*, infraohjelmointi_api/**/migrations/**/*, fix-database.sql
```

Comment:
Does this sonar.exclusions thing skip Sonar Cloud checks for the matching files?

Reply:
It does. The idea behind this were the file will be run only once when merging new features to dev and prod. But the script need to be run when creating local environment, because imported files includes incorrect data. That's why it might be better not to be included in this list.

I will remove this change.